Tests/IGListStackSectionControllerTests.m

```diff
@@ -85,8 +85,9 @@ - (void)test_whenInitializingStack_thatNumberOfItemsMatches {
     section4.items = 1;
 
     IGListStackedSectionController *stack = [[IGListStackedSectionController alloc] initWithSectionControllers:@[section1, section2, section3, section4]];
-
-    XCTAssertEqual([stack numberOfItems], 6);
+    [stack performBatchAnimated:false updates:^{} completion:^(BOOL finished) {
+        XCTAssertEqual([stack numberOfItems], 6);
```

Comment:
These tests use [IGListAdapterUpdater](https://github.com/Instagram/IGListKit/blob/master/Tests/IGListStackSectionControllerTests.m#L57) which asynchronously collect updates during a runloop turn, so this assert wont execute w/out an expectation+wait setup. Here's an [example](https://github.com/Instagram/IGListKit/blob/master/Tests/IGListStackSectionControllerTests.m#L766-L776). Same goes for all the asserts in this change that got moved into the completion block.

A really easy way to check that the block executes is:

```objc
__block BOOL executed = NO;
[adapter performUpdates:^{
  executed = YES;
}];
XCTAssertTrue(executed);
```